easy/25.js

```diff
@@ -1,12 +1,12 @@
-function isZipCode(zipcode){
+function isZipCode(zipcode) {

```

Comment:
Could you explain using this questionmark?

Reply:
i can't, i'm beginner with regex and don't understand fully this. I used code which I found on the internet. After tests, yes without questionmark also working.